Could not resolve @ng-toolkit/universal relative to app.module.ts..

## I'm submitting a...
<!-- Check one of the following options with "x" -->
<pre><code>
[x] Bug report 
[ ] Feature request
[ ] Documentation issue or request
</code></pre>

## In package
<pre><code>
[ ] @ng-toolkit/init
[ ] @ng-toolkit/serverless
[x] @ng-toolkit/universal
[ ] @ng-toolkit/pwa
[ ] @ng-toolkit/firebug
</code></pre>

## Current behavior
running `ng add @ng-toolkit/universal` will only create `utils` and `serverless` folders inside `node_modules/@ng-toolkit` on an existing project.


## Expected behavior
<!-- Describe what the desired behavior would be. -->
running  `ng add @ng-toolkit/universal` should install all dependencies required in `app.module.ts`


## Minimal reproduction of the problem with instructions

1. run `ng add @ng-toolkit/universal` on an existing angular 6 project
2. run `npm run build:prod`
3. This error will pop up:
```
ERROR in app/app.module.ts(121,1): Error during template compile of 'AppModule'
  Could not resolve @ng-toolkit/universal relative to /project/path/src/app/app.module.ts..
src/app/pages/post/post.component.ts(1,24): error TS2307: Cannot find module '@ng-toolkit/universal'.
src/app/layouts/header-nav/header-nav.component.ts(1,24): error TS2307: Cannot find module '@ng-toolkit/universal'.
src/app/layouts/slider-home/slider-home.component.ts(1,24): error TS2307: Cannot find module '@ng-toolkit/universal'.
src/app/layouts/header-nav-post/header-nav-post.component.ts(1,24): error TS2307: Cannot find module '@ng-toolkit/universal'.
src/app/directives/scrollable.directive.ts(1,24): error TS2307: Cannot find module '@ng-toolkit/universal'.
src/app/app.module.ts(1,36): error TS2307: Cannot find module '@ng-toolkit/universal'.
```

## Example repository
I cannot provide this because company privacy policies, but i will send individual resources as requested

## What is the motivation / use case for changing the behavior?
The simpliest workaround is to manually installing it with `npm install --save @ng-toolkit/universal`, but i think you should know this is happening.


## Environment

<pre><code>
Angular CLI: 6.2.3
Node: 8.11.3
OS: linux x64
Angular: 6.1.9
... animations, common, compiler, compiler-cli, core, forms
... http, language-service, platform-browser
... platform-browser-dynamic, platform-server, router

Others:
<!-- Anything else relevant?  Operating system version, IDE, package manager, HTTP server, ... -->
Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.7.5
@angular-devkit/build-angular     0.7.5
@angular-devkit/build-optimizer   0.7.5
@angular-devkit/build-webpack     0.7.5

@angular-devkit/core              0.7.5
@angular-devkit/schematics        0.8.3
@angular/cdk                      6.4.7
@angular/cdk-experimental         6.2.0
@angular/cli                      6.2.3
@angular/fire                     5.0.2
@angular/material                 6.4.7
@ngtools/webpack                  6.1.5
@schematics/angular               0.8.3
@schematics/update                0.8.3
rxjs                              6.3.3
typescript                        2.9.2
webpack                           4.9.2

</code></pre>

--- a/.gitignore
+++ b/.gitignore
@@ -2,3 +2,5 @@
 .vscode/
 *.iml
 dist/
+verdacio_output
+scripts/storage/*

--- a/schematics/_utils/package.json
+++ b/schematics/_utils/package.json
@@ -1,7 +1,7 @@
 {
   "author": "Maciej Treder <contact@maciejtreder.com>",
   "name": "@ng-toolkit/_utils",
-  "version": "8.0.2",
+  "version": "8.0.3",
   "description": "Angular Universal PWA boilerplate for serverless environment.",
   "repository": {
     "type": "git",

--- a/schematics/firebug/package.json
+++ b/schematics/firebug/package.json
@@ -1,7 +1,7 @@
 {
   "author": "Maciej Treder <contact@maciejtreder.com>",
   "name": "@ng-toolkit/firebug",
-  "version": "8.0.2",
+  "version": "8.0.3",
   "description": "Add Firebug lite to your Angular project.",
   "repository": {
     "type": "git",
@@ -32,7 +32,7 @@
     "@angular-devkit/core": "^8.3.3",
     "@angular-devkit/schematics": "^8.3.3",
     "@bugsnag/js": "^6.4.0",
-    "@ng-toolkit/_utils": "8.0.2",
+    "@ng-toolkit/_utils": "8.0.3",
     "@schematics/angular": "^8.3.3"
   },
   "devDependencies": {

--- a/schematics/init/package.json
+++ b/schematics/init/package.json
@@ -11,7 +11,7 @@
     "build": "tsc -p tsconfig.json",
     "test": "npm run build && jasmine src/**/*_spec.js",
     "postinstall": "node postinstall.js",
-    "prepublish": "npm test",
+    "prepare": "npm test",
     "ci-publish": "ci-publish"
   },
   "keywords": [

--- a/schematics/init/package.json
+++ b/schematics/init/package.json
@@ -1,7 +1,7 @@
 {
   "author": "Maciej Treder <contact@maciejtreder.com>",
   "name": "@ng-toolkit/init",
-  "version": "8.0.2",
+  "version": "8.0.3",
   "description": "Angular Universal PWA boilerplate for serverless environment.",
   "repository": {
     "type": "git",
@@ -32,9 +32,9 @@
     "@angular-devkit/core": "^8.3.3",
     "@angular-devkit/schematics": "^8.3.3",
     "@bugsnag/js": "^6.4.0",
-    "@ng-toolkit/_utils": "8.0.2",
-    "@ng-toolkit/firebug": "8.0.2",
-    "@ng-toolkit/serverless": "8.0.2",
+    "@ng-toolkit/_utils": "8.0.3",
+    "@ng-toolkit/firebug": "8.0.3",
+    "@ng-toolkit/serverless": "8.0.3",
     "@schematics/angular": "^8.3.3"
   },
   "devDependencies": {

--- a/schematics/pwa/package.json
+++ b/schematics/pwa/package.json
@@ -2,7 +2,7 @@
   "author": "Maciej Treder <contact@maciejtreder.com>",
   "name": "@ng-toolkit/pwa",
   "main": "dist/index.js",
-  "version": "8.0.2",
+  "version": "8.0.3",
   "description": "Extension for @angular/pwa - adds server-side rendering fixes and update mechanism",
   "repository": {
     "type": "git",
@@ -37,10 +37,10 @@
   },
   "dependencies": {
     "@bugsnag/js": "^6.4.0",
-    "@ng-toolkit/_utils": "8.0.2"
+    "@ng-toolkit/_utils": "8.0.3"
   },
   "devDependencies": {
-    "@ng-toolkit/universal": "^8.0.1",
+    "@ng-toolkit/universal": "^8.0.3",
     "@angular-devkit/build-angular": "^0.803.4",
     "@angular-devkit/build-ng-packagr": "^0.803.4",
     "@angular-devkit/core": "^8.3.3",

--- a/schematics/pwa/publish.sh
+++ b/schematics/pwa/publish.sh
@@ -1,1 +1,4 @@
-npm run ci-publish
+# Move to generated production folder
+cd dist
+# Run package ci-publish script
+./../node_modules/ci-publish/bin/ci-publish.js 

--- a/schematics/pwa/publish_verdaccio.sh
+++ b/schematics/pwa/publish_verdaccio.sh
@@ -1,1 +1,4 @@
+# Move to generated production folder
+cd dist
+# Run publish script
 npm publish --registry http://localhost:4873

--- a/schematics/serverless/package.json
+++ b/schematics/serverless/package.json
@@ -1,7 +1,7 @@
 {
   "author": "Maciej Treder <contact@maciejtreder.com>",
   "name": "@ng-toolkit/serverless",
-  "version": "8.0.2",
+  "version": "8.0.3",
   "description": "Angular Universal PWA boilerplate for serverless environment.",
   "repository": {
     "type": "git",
@@ -32,7 +32,7 @@
     "@angular-devkit/core": "^8.3.3",
     "@angular-devkit/schematics": "^8.3.3",
     "@bugsnag/js": "^6.4.0",
-    "@ng-toolkit/_utils": "8.0.2",
+    "@ng-toolkit/_utils": "8.0.3",
     "@schematics/angular": "^8.3.3"
   },
   "devDependencies": {

--- a/schematics/universal/package.json
+++ b/schematics/universal/package.json
@@ -2,7 +2,7 @@
   "author": "Maciej Treder <contact@maciejtreder.com>",
   "name": "@ng-toolkit/universal",
   "main": "dist/index.js",
-  "version": "8.0.2",
+  "version": "8.0.3",
   "description": "Adds Angular Universal support for any Angular CLI project",
   "repository": {
     "type": "git",
@@ -38,7 +38,7 @@
     "@angular-devkit/core": "^8.3.3",
     "@angular-devkit/schematics": "^8.3.3",
     "@bugsnag/js": "^6.4.0",
-    "@ng-toolkit/_utils": "8.0.2",
+    "@ng-toolkit/_utils": "8.0.3",
     "@nguniversal/express-engine": "^8.1.1",
     "@schematics/angular": "^8.3.3"
   },
@@ -50,8 +50,8 @@
     "@angular/compiler": "^8.2.6",
     "@angular/compiler-cli": "^8.2.6",
     "@angular/core": "^8.2.6",
-    "@ng-toolkit/pwa": "8.0.2",
-    "@ng-toolkit/serverless": "8.0.2",
+    "@ng-toolkit/pwa": "8.0.3",
+    "@ng-toolkit/serverless": "8.0.3",
     "@types/jasmine": "^3.4.0",
     "@types/node": "^12.7.4",
     "ci-publish": "^1.3.1",

--- a/schematics/universal/publish.sh
+++ b/schematics/universal/publish.sh
@@ -1,1 +1,4 @@
-npm run ci-publish
+# Move to generated production folder
+cd dist
+# Run package ci-publish script
+./../node_modules/ci-publish/bin/ci-publish.js 

--- a/schematics/universal/publish_verdaccio.sh
+++ b/schematics/universal/publish_verdaccio.sh
@@ -1,1 +1,4 @@
+# Move to generated production folder
+cd dist
+# Run publish script
 npm publish --registry http://localhost:4873

--- a/scripts/deploy.sh
+++ b/scripts/deploy.sh
@@ -2,7 +2,7 @@ npm install -g @angular/cli
 
 cd schematics
 
-CATALOGS=(_utils serverless universal pwa firebug)
+CATALOGS=(_utils serverless universal pwa firebug init)
 
 exitStatus=0
 

--- a/scripts/deploy.sh
+++ b/scripts/deploy.sh
@@ -1,8 +1,7 @@
-npm install -g @angular/cli
-
-cd schematics
-
 CATALOGS=(_utils serverless universal pwa firebug init)
+
+npm install -g @angular/cli
+cd schematics
 
 exitStatus=0
 

--- a/scripts/deploy_verdaccio.sh
+++ b/scripts/deploy_verdaccio.sh
@@ -1,4 +1,4 @@
-CATALOGS=(_utils serverless pwa universal firebug init)
+CATALOGS=(_utils serverless universal pwa firebug init)
 echo "//localhost:4873/:_authToken=\"CjmKyL6UDkX6FDpNnP64fw==\"" >>~/.npmrc
 
 npm install -g @angular/cli
@@ -23,9 +23,6 @@ for i in "${CATALOGS[@]}"; do
     cd $i
     npm install --verbose
     printf '\e[1;32m--- NPM INSTALL COMPLETED FOR %s ---\e[0m\n' "$i"
-    if [ "$i" != "_utils" ]; then
-        ls node_modules/@ng-toolkit/_utils
-    fi
     if ./publish_verdaccio.sh; then
         printf '\e[1;33m*** %s PASSED ***\e[0m\n' "$i"
     else

--- a/scripts/deploy_verdaccio.sh
+++ b/scripts/deploy_verdaccio.sh
@@ -1,4 +1,4 @@
-CATALOGS=(_utils serverless pwa universal firebug)
+CATALOGS=(_utils serverless pwa universal firebug init)
 echo "//localhost:4873/:_authToken=\"CjmKyL6UDkX6FDpNnP64fw==\"" >>~/.npmrc
 
 npm install -g @angular/cli

